lib/internal/streams/duplexpair.js

```diff
@@ -39,13 +38,13 @@ class DuplexSocket extends Duplex {
   }
 }
 
-class DuplexPair {
+class DuplexPair extends Array {
```

Comment:
I would prefer if this was not a class a tall, but rather a factory and it did not inherit from `Array`.

Reply:
@awwright you might have missed this.